src/main/java/com/blackducksoftware/integration/fortify/batch/step/BlackDuckFortifyPhoneHomeStep.java

```diff
@@ -72,18 +70,13 @@ public ExitStatus afterStep(StepExecution stepExecution) {
     }
 
     @Override
-    public RepeatStatus execute(StepContribution contribution, ChunkContext chunkContext) throws Exception {
+    public RepeatStatus execute(final StepContribution contribution, final ChunkContext chunkContext) throws Exception {
         logger.info("Started Phone Home step");
 
-        PhoneHomeService phoneHome = hubServices.getPhoneHomeDataService();
-        PhoneHomeRequestBodyBuilder phoneHomeReq = phoneHome.createInitialPhoneHomeRequestBodyBuilder();
-        phoneHomeReq.setSource(PhoneHomeSource.ALLIANCES);
-        phoneHomeReq.setThirdPartyName(ThirdPartyName.FORTIFY_SSC);
-        phoneHomeReq.setThirdPartyVersion("N/A");
-        phoneHomeReq.setPluginVersion(propertyConstants.getPluginVersion());
-
+        final PhoneHomeService phoneHome = hubServices.createPhoneHomeDataService();
+        final Builder phoneHomeReq = phoneHome.createInitialPhoneHomeRequestBodyBuilder("fortify-ssc", propertyConstants.getPluginVersion());
+        phoneHomeReq.addToMetaData("Source", "Alliance Integrations");
```

Comment:
Are there still static variables (PhoneHomeSource and ThirdPartyName) that encapsulate "Alliance Integrations" and "fortify-ssc" so there are no "magic" constants in the code?